docs/decisions/governance/code-of-conduct_DECISION.md

```diff
@@ -8,29 +8,54 @@
   - [@seporterfield](https://github.com/seporterfield)
   - [@Monstarrrr](https://github.com/Monstarrrr) 4 / 5
 
-- **Last updated**: 26-05-2024
-
-**Technical Story**: [related issue URL] <!-- optional -->
+- **Technical Story**:
+  - [Your Code of Conduct - (opensource.guide)](https://opensource.guide/code-of-conduct/)
+- **Last updated**: 27-05-2024
 
 ## Context and Problem Statement
 
-The impact of the following is low as we do not have a community yet.
+The impact of the code of conduct is low as we do not have a community yet.
 
-## Decision Drivers <!-- optional -->
+## Decision Drivers
 
-- [driver 1, e.g., a force, facing concern, …]
-- [driver 2, e.g., a force, facing concern, …]
-- … <!-- numbers of drivers can vary -->
+1. **Accessibility**: Ensure the document is easily accessible within the project repository and available in multiple languages if necessary.
+2. **Specific Examples**: Includes specific examples of both acceptable and unacceptable behaviors to eliminate ambiguity.
+3. **Concise**: Easy to read and remember, focuses on the most important points.
+4. **Positive Tone**: Describes guidelines in a positive way.
+5. **Reporting Process**: Defines steps for reporting violations and provides multiple channels for doing so, ensures the process is straightforward and confidential.
+6. **Defined Consequences**: Specify the consequences of violating the Code of Conduct to ensure consistency and transparency in enforcement.
+7. **Inclusivity**: Emphasizes the importance of inclusivity and respect for all community members.
+8. **Community Input**: Involves the community in the development and revision of the Code to ensure it reflects shared values and concerns.
 
 ## Decision Outcome
 
-**Chosen option**: "Slightly modified default github `CODE_OF_CONDUCT.md`"  
-**Reason(s)**: The most basic code of conduct should suffice especially since we don't have a community yet anyways.
-
-### Positive Consequences <!-- optional -->
+**Chosen option**:  
+Slightly modified default github `CODE_OF_CONDUCT.md`:
+
+1. English only
+2. Examples of both standard and unacceptable behavior
+3. - Our pledge
+   - Our standards
+   - Enforcement
+   - Consequences
+   - Attribution
+4. Tone is neutral not positive
+5. No steps provided to report violations
+6. Consequences are defined
+   - Correction
+   - Warning
+   - Temporary ban
+   - Permanent ban
+7. Inclusivity emphasized multiple times throuhought the document
+8. Not community input options provided
+
+**Reason(s)**: A basic code of conduct (heavily inspired by the default github code of conduct) should suffice especially since we don't have a community yet anyways.
```

Comment:
maybe remove this? ["should" ](https://www.psychologytoday.com/us/blog/the-path-passionate-happiness/202205/why-no-good-comes-the-word-should)

Reply:
That's an interesting point in general, although there exists specifications for the use of such word in formal documents, which I believe is a different context than the link you provided.  

I would propose to follow the meanings as described in [RFC 2119](https://www.ietf.org/rfc/rfc2119.txt):  

> SHOULD: This word, or the adjective "RECOMMENDED", mean that there
   may exist valid reasons in particular circumstances to ignore a
   particular item, but the full implications must be understood and
   carefully weighed before choosing a different course.  

Basically in this example, a basic code of conduct does suffice in the current circumstances, but that could change if the circumstances were to change (e.g. new contributors to the project joined).

Reply:
That's interesting. I've never seen that definition. I don't agree with anything that suggests we "have to" do anything https://youtu.be/jyqGGAn2R1E?feature=shared, such as "must" in your should definition and "should" in the way I was perceiving it earlier.

Everything else looks fine.